app/src/main/java/org/fossasia/susi/ai/activities/ImportantMessages.java

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.susi.ai.activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.RecyclerView;
```

Comment:
I have added RecyclerView to the activity which a user wants to open when he/she wants to find messages marked as important. the thing left is show the messages in the activity. will commit it.